test/Blockfrost.Api.Tests/dat/01/create.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+#
+# Script to create the test vector
+SCRIPT=$(readlink -f "$0")
+VDIR=$(dirname $SCRIPT)
+DDIR=$(dirname $VDIR)
+AMOUNT=5000
+MAGIC=1097911063
```

Comment:
@mmahut @eddex If your bash skills are better than mine feel free to improve this.

Reply:
and below...